ambry-frontend/src/main/java/com.github.ambry.frontend/AmbrySecurityService.java

```diff
@@ -130,7 +133,8 @@ public void processResponse(RestRequest restRequest, RestResponseChannel respons
       if (operationOrBlobId.startsWith("/")) {
         operationOrBlobId = operationOrBlobId.substring(1);
       }
-      if (blobInfo == null && !restRequest.getRestMethod().equals(RestMethod.OPTIONS)) {
+      if (blobInfo == null && !restRequest.getRestMethod().equals(RestMethod.OPTIONS) && !restRequest.getRestMethod()
+          .equals(RestMethod.PUT)) {
         if (!operationOrBlobId.equals(Operations.GET_SIGNED_URL)) {
```

Comment:
Kind of orthogonal, but I believe we ignore the URI (for now) on POST requests so this check wouldn't trigger if someone attempted a `POST /signedUrl`

Reply:
but that's not a valid op - only `GET /signedUrl` is. Something I'm missing?

Reply:
Yep, it's not a valid operation. I don't think there is any code in `PostBlobHandler` checking the path on POST requests, so someone could send a `POST /signedUrl` request (and it would be handled the same way as a `POST /` request. However, when the code gets to this `blobInfo != null` check will not get triggered. This isn't a huge problem since the code will just throw a NPE a few lines down. 

Anyways, `AmbryBlobStorageService` is probably the place to check that the path on a POST request is `/`